unit-tests/shared/src/test/require-jdk9/org/scalanative/testsuite/javalib/util/ArraysOfAnyValTestOnJDK9.scala.gyb

```diff
@@ -0,0 +1,1467 @@
+package org.scalanative.testsuite.javalib.util
+%{
+## ArraysOfAnyValTestOnJDK9.scala.gyb
+##
+##
+## To generate this file's output manually, execute the python script
+## 'scripts/gyb.py' under the project root. For example, from the project root:
+##
+##   scripts/gyb.py \
+##     ${thisDirectory}/ArraysOfAnyValTestOnJDK9.scala.gyb \
+##     --line-directive '' \
+##     -o ${thisDirectory}/ArraysOfAnyValTestOnJDK9.scala
+##
+## Then manually run scripts/scalafmt on the output file.
+## The generated code is formatted but not to the exact standard used by CI.
+## Allowing such slight format differences, which scalafmt will easily
+## touch-up, makes this file much simpler.
```

Comment:
Similary as with JDK9Methods template this one should be added to `scripts/gyb_all.sh`